libraries/azurerm_api_management.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'azurerm_resource'
+
+class AzurermApiManagement < AzurermSingularResource
+  name 'azurerm_api_management'
+  desc 'Verifies settings for an Azure Api Management Service'
+  example <<-EXAMPLE
+    describe azurerm_api_management(resource_group: 'rg-1', api_management_name: 'apim01') do
+      it { should exist }
+    end
+  EXAMPLE
+
+  ATTRS = %i(
+    id
+    name
+    location
+    type
+    properties
+  ).freeze
+
+  attr_reader(*ATTRS)
+
+  def initialize(resource_group: nil, api_management_name: nil)
+    api_management = management. api_management(resource_group, api_management_name)
```

Comment:
There is a space after `management`. 
`api_management = management.api_management(resource_group, api_management_name)`